app/models/cphc_facility.rb

```diff
@@ -47,6 +47,52 @@ def self.create_subcenter_from_mapping(mapping)
     end
   end
 
+  def self.create_chc_from_row(row)
+    CphcFacility.find_or_create_by!(cphc_facility_id: row[:cphc_facility_id]) do |facility|
+      facility.facility_id = nil
+      facility.cphc_facility_id = row[:cphc_facility_id]
+      facility.cphc_facility_name = row[:cphc_facility_name]
+      facility.cphc_district_id = row[:cphc_district_id]
+      facility.cphc_district_name = row[:cphc_district_name]
+      facility.cphc_taluka_id = row[:cphc_taluka_id]
+      facility.cphc_taluka_name = row[:cphc_taluka_name]
+      facility.cphc_user_details = nil
+      facility.cphc_facility_type = "CHC"
+      facility.cphc_facility_type_id = OneOff::CphcEnrollment::FACILITY_TYPE_ID["CHC"]
+      facility.cphc_location_details = {
```

Comment:
We don't need to populate location details for PHCs and HWCs?

Reply:
For PHCs and HWCs we look up the location from CPHC mappings. For CHCs and DHs we had to change this to accommodate for dummy locations.

Reply:
Ah makes sense. 👍